package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
Let's remove the packages, both. You aren't adding any new JS module. Or, if you do, you could put in in our scripts folder.

Reply:
Done!